core/src/com/cloud/agent/api/storage/CopyVolumeCommand.java

```diff
@@ -75,4 +76,12 @@ public boolean toSecondaryStorage() {
     public String getVmName() {
         return vmName;
     }
+
+    public String getNfsVersion() {
+        return nfsVersion;
+    }
+
+    public void setNfsVersion(String nfsVersion) {
+        this.nfsVersion = nfsVersion;
+    }
```

Comment:
@jburwell Can you click on the OSPF commit for review, somehow during rebase non ospf commits also got included. I will try to clean them up.